centos_ci.sh

```diff
@@ -371,7 +371,11 @@ else
   elif [[ "$JOB_NAME" = "minishift-release" ]]; then
     perform_release $RSYNC_PASSWORD;
   elif [[ "$JOB_NAME" = "minishift-nightly" ]]; then
-    perform_nightly;
+    perform_nightly "b2d";
+  elif [[ "$JOB_NAME" = "minishift-nightly-minikube" ]]; then
+    perform_nightly "minikube";
```

Comment:
I didn't see any job exist https://github.com/minishift/minishift-ci-jobs/blob/master/minishift-ci-index.yaml so first we should update our JJB file and then move forward. cc @budhrg